openstack/networking/v2/extensions/security/rules/requests.go

```diff
@@ -150,6 +150,25 @@ func Create(ctx context.Context, c *gophercloud.ServiceClient, opts CreateOptsBu
 	return
 }
 
+// CreateBulk is an operation which adds new security group rules and associates them
+// with an existing security group (whose ID is specified in CreateOpts).
+func CreateBulk(ctx context.Context, c *gophercloud.ServiceClient, opts []CreateOpts) (r CreateResult) {
+	createOpts := make([]map[string]any, len(opts))
```

Comment:
Alternatively, we can modify [gophercloud.BuildRequestBody()](https://github.com/gophercloud/gophercloud/blob/de76eeb/params.go#L35) to accept an array/slice. Perhaps this would be more user friendly? Do we have a preference?

The nice thing if we decide to make gophercloud.BuildRequestBody() accept slices or arrays, is that it does not require changing the signature function, allowing us to backport the change.

Reply:
I've moved the logic in BuildRequestBody(), as it makes more sense to me. PTAL

https://github.com/gophercloud/gophercloud/compare/324a1105f8e764e30e5483bd95176f347dfc07a9..b1f8bbb05730e871f8d06bb6de2e28fcb50da3d5